docs/user/security/securing-kibana.asciidoc

```diff
@@ -59,13 +59,14 @@ For more information, see <<security-settings-kb,Security Settings in {kib}>>.
 . Optional: Set a timeout to expire idle sessions. By default, a session stays
 active until the browser is closed. To define a sliding session expiration, set
 the `xpack.security.session.idleTimeout` property in the `kibana.yml`
-configuration file. The idle timeout is specified in milliseconds. For example,
-set the idle timeout to 600000 to expire idle sessions after 10 minutes:
+configuration file. The idle timeout is formatted as a duration of
+`<count>[ms|s|m|h|d|w|M|Y]` (e.g. '70ms', '5s', '3d', '1Y'). For example, set
+the idle timeout to expire idle sessions after 10 minutes:
 +
 --
 [source,yaml]
 --------------------------------------------------------------------------------
-xpack.security.session.idleTimeout: 600000
+xpack.security.session.idleTimeout: "10m"
```

Comment:
__nit:__ I believe `"` aren't required (and in example below as well)

Reply:
That's correct, but all other examples of strings in the asciidocs seem to have double quotes surrounding them, so I added them here for consistency.

Reply:
Ah, I see, good then :+1: